WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/ProductSelectorViewModelTests.swift

```diff
@@ -792,15 +833,16 @@ final class ProductSelectorViewModelTests: XCTestCase {
         }
 
         // When
-        viewModel.filters = filters
+        viewModel.updateFilters(filters)
         viewModel.searchTerm = ""
         try await Task.sleep(nanoseconds: searchDebounceTime)
 
         // Then
-        assertEqual(filteredStockStatus, filters.stockStatus)
-        assertEqual(filteredProductType, filters.productType)
-        assertEqual(filteredProductStatus, filters.productStatus)
-        assertEqual(filteredProductCategory, filters.productCategory)
+        let currentFilters = viewModel.retrieveUpdatedFilters()
+        assertEqual(filteredStockStatus, currentFilters.stockStatus)
+        assertEqual(filteredProductType, currentFilters.productType)
+        assertEqual(filteredProductStatus, currentFilters.productStatus)
+        assertEqual(filteredProductCategory, currentFilters.productCategory)
```

Comment:
❓ if I understand the test case correctly, we want to make sure the `filtered*` parameters match the `filters` in the `updateFilters` function? I'm not sure if I missed something 🤔 

```suggestion
        assertEqual(filteredStockStatus, filters.stockStatus)
        assertEqual(filteredProductType, filters.productType)
        assertEqual(filteredProductStatus, filters.productStatus)
        assertEqual(filteredProductCategory, filters.productCategory)
```

Reply:
That's right, we can remove the `currentFilters` declaration. Changes on https://github.com/woocommerce/woocommerce-ios/pull/9387/commits/e389ebf88b17efa4b10f4569d0abb9f41cacac55